dashboard/scripts_restructure.rb

```diff
@@ -0,0 +1,26 @@
+#
+# This will generate two new files:
```

Comment:
Ultimately probably want this file and changes to scripts.en.yml in a separate PR (and this file may not belong in the repo)